README.md

```diff
@@ -14,7 +14,7 @@
 
 ## Usage
 
-<img src="resources/pack-build.gif" width="600px" />
+<img src="https://github.com/buildpacks/pack/raw/main/resources/pack-build.gif" width="600px" />
```

Comment:
I'd love to hear another suggestion if this isn't the best idea! I could also include the resource in the package, but that doesn't seem like the nicest solution, it would just needlessly clog the chocolatey package.

Reply:
TLDR; Might be worth NOT posting a README.md

I'm looking around at some other packages and how this file would be used and I have a few concerns:

1. I haven't found that this README.md is used anywhere by another project.
2. How would this README be presented? If it's not and it's only listed in the `files` then what's its purpose?
3. The README.md is currently tailored to be hosted on GH. Some package registries (such as npm) do a good job about resolving links but chocolatey doesn't.

Based on the above and the added complexity/concerns, I would suggest that the `<description>` is sufficient from an end-user's perspective. I'm assuming it can then refer to (link) to additional details.

Reply:
I'm pretty sure that this is how it gets displayed, which seems pretty visible to me: https://chocolatey.org/packages/buildpacks#description

Reply:
Is there a way to confirm? If I download the `nupkg`. The README.md isn't there. Instead the content is in the `<description>` field:

![image](https://user-images.githubusercontent.com/475559/86178465-02ca0c00-baee-11ea-987b-70ec2595cf17.png)
![image](https://user-images.githubusercontent.com/475559/86178507-12e1eb80-baee-11ea-86f6-e59a58386f0d.png)

Reply:
To clarify what this change is about - above, we read in the `README` and putting it into the `description` tag. We don't publish a separate `README` file in the `nupkg`.